addons/block_code/ui/blocks/block/block.gd

```diff
@@ -103,42 +86,8 @@ func disconnect_signals():
 		drag_started.disconnect(c.callable)
 
 
-func update_resources(undo_redo: EditorUndoRedoManager):
```

Comment:
So the new architecture makes it imposible to reuse the resources? This probably makes the resource IDs change constantly. I was hoping that we were going to drop that from William's PR. That is, have a nice AST for very well proven code generation, but keep passing the resources around.

Reply:
Ah, it does regenerate IDs frequently. I hadn't paid attention to that part of the changes. I'll see if I can keep that part.